lib/import/context/import-context-fetcher.ts

```diff
@@ -166,7 +166,7 @@ export async function importContextFetcherAsync(config: ImportContextConfig) {
 
                     if (!latestLanguageVariant) {
                         // there is neither published or draft version as latest version does not exist at all
-                        return undefined;
+                        return '404';
```

Comment:
That's because when you fetch some data, they might not exist in the target environment and that is fine. In some places I wanted to distinguish between failed request, 404 request and a valid request ;) Using undefined made it unclear as to what state the request is in.